Documentation/guides/WindowsOnArm64.md

```diff
@@ -39,8 +39,7 @@ This downloads the latest files in `C:\Program Files\dotnet\sdk-manifests`.
 
 The Mono workloads from dotnet/runtime need a small fix.
 
-Open `C:\Program Files\dotnet\sdk-manifests\7.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.json`
-in your favorite text editor.
+Open each `WorkloadManifest.json` under the folder tree `C:\Program Files\dotnet\sdk-manifests` in your favorite text editor and make the following changes:
```

Comment:
A problem with this suggestion is that it might be (is?) "too much"; consider:

```
# find /mnt/c/Program\ Files/dotnet/sdk-manifests/ -iname WorkloadManifest.json | wc -l
32

# find /mnt/c/Program\ Files/dotnet/sdk-manifests/ -iname WorkloadManifest.json
./6.0.100/microsoft.net.sdk.android/WorkloadManifest.json
./6.0.100/microsoft.net.sdk.ios/WorkloadManifest.json
./6.0.100/microsoft.net.sdk.maccatalyst/WorkloadManifest.json
…
./7.0.100/microsoft.net.sdk.android/WorkloadManifest.json
./7.0.100/microsoft.net.sdk.ios/WorkloadManifest.json
./7.0.100/microsoft.net.sdk.maccatalyst/WorkloadManifest.json
./7.0.100/microsoft.net.sdk.macos/WorkloadManifest.json
…
./7.0.100/microsoft.net.workload.mono.toolchain.net6/WorkloadManifest.json
./7.0.100/microsoft.net.workload.mono.toolchain.net7/WorkloadManifest.json
```

I have *32* `WorkloadManifest.json` files under `C:\Program Files\dotnet\sdk-manifests`.  Should I really edit *all 32* of them?  (I imagine ***no***?)

Assuming I *shouldn't* edit all 32 of them, which ones *should* I edit?

We don't want people to screw up their installs.